packages/valory/skills/abstract_round_abci/skill.yaml

```diff
@@ -9,8 +9,8 @@ fingerprint:
   README.md: QmYNXPppPSnW61ipqKNH8pyMh8qzTERwQaducYYq1dnBJb
```

Comment:
The new param needs to be set to null here otherwise it's not configurable (overrides only possible for existing keys)

Reply:
I have seen that this yaml specifies no parameters but all the other ones for other skills do, so for coherence I'll add the param in the same way.